server/router/index.js

```diff
@@ -1,72 +1,20 @@
 const db = require('../database');
 
-const getRecordById = (req, res, next) => {
-  db.postgresql.getRecordById(req.params.id)
-    .then((data) => {
-      // console.log('getRecordById data:', data);
-      res.status(200);
-      res.json(data.rows);
-    })
-    .catch((error) => {
-      console.log('getRecordById error:', error);
-      res.status(500).end('Something went wrong. Please try again later.');
-    });
-};
+const get = (query) => (
```

Comment:
Hi Elvin, you probably want to add some comments in these files. Both for yourself and for other developers who may want to read this. :)